src/openforms/submissions/api/validation.py

```diff
@@ -5,65 +5,119 @@
 to the frontend.
```

Comment:
I guess this TODO cannot be removed right?Your current work is not entirely solving the problem?

Reply:
removed!